pod/perlfunc.pod

```diff
@@ -4359,12 +4370,6 @@ and attributes are handled using the L<attributes> pragma, or starting
 from Perl 5.8.0 also via the L<Attribute::Handlers> module.  See
 L<perlsub/"Private Variables via my()"> for details.
 
-Note that with a parenthesised list, L<C<undef>|/undef EXPR> can be used
```

Comment:
was this paragraph intentionally moved?

Reply:
(I think the new location is better, but just checking as it's not really relevant)

Reply:
Intentional on the basis that (1) the paragraphs for both functions then appear in a more similar-seeming order than before (2) the "gotchas"/see-links-for-more-info paragraphs come last, which I think aids comprehension - especially for newcomers.